crates/bitwarden/src/platform/generate_fingerprint.rs

```diff
@@ -45,7 +45,7 @@ pub(crate) fn generate_user_fingerprint(
         .ok_or("Missing private key")?;
 
     let public_key = private_key.to_public_der()?;
-    let fingerprint = fingerprint(&fingerprint_material, &public_key)?;
+    let fingerprint = fingerprint(&fingerprint_material, public_key.expose())?;
```

Comment:
Should we update fingerprint to take the sensitive value directly? Then we would only need to use expose in a single place.

Reply:
I changed public keys to not be sensitive.